### DISCUSSION TITLE
Skipping Bentley login

### DISCUSSION THREAD

**Discussion by kardoka** [iTwin/itwinjs-core#4859] [Q&A] [answered]
Greetings and happy New Year everyone!

I would like to let users use my app without having their own Bentley accounts, kind of how almost anybody can interact with the iTwin.js samples.

Could this be achieved? If so, then how? As an idea, could it be possible to pass in the login credentials from something like a .env file?

**Comment by aruniverse** [comment, marked as answer]
Hi,

Please take a look at: https://github.com/calebmshafer/itwin-viewer-proxy

**Reply by kardoka** [reply]
Thank you, I will check it out.
